src/main/kotlin/org/rust/lang/utils/RsDiagnostic.kt

```diff
@@ -1249,6 +1249,16 @@ sealed class RsDiagnostic(
         )
     }
 
+    class DeriveAttrUnsupportedItem(element: PsiElement,
```

Comment:
Let's also provide quick fix to remove attribute
See `org.rust.ide.annotator.fixes.RemoveAttrFix`